### PR TITLE
make track selection work with run3 simulations

### PR DESCRIPTION
**Pull request by mario-krueger**

### CODE REVIEW

File: Common/Core/TrackSelection.h [hunk 1]
```diff
@@ -77,37 +78,54 @@ class TrackSelection
   template <typename T>
   bool IsSelected(T const& track, const TrackCuts& cut)
   {
+    bool isRun2 = track.trackType() == o2::aod::track::Run2Track || track.trackType() == o2::aod::track::Run2Tracklet;
```
Comment (jgrosseo) [inline Common/Core/TrackSelection.h:81]: Why do you need to evaluate this on each track?

Comment (jgrosseo) [reply Common/Core/TrackSelection.h:81]: You want to autodetect this setting?

Comment (mario-krueger) [reply Common/Core/TrackSelection.h:81]: how can this can be implemented?

**Comment by jgrosseo** [comment]
Until we have the meta data reading, I propose you keep the run check as you did it now.
This PR https://github.com/AliceO2Group/AliceO2/pull/7075 fixes the TrackType, so maybe we can get rid of the work around, right away. What do you think?

**Comment by jgrosseo** [comment]
Merging despite further changes are coming.

**Pull request merged**